tests/apps/test_sc_remote.py

```diff
@@ -65,7 +66,6 @@ def versions():
 
 ###########################
 @pytest.mark.quick
-@pytest.mark.timeout(30)
```

Comment:
Was this causing issues before?

Reply:
no, the default timeout is 120s, no reason to make it shorter